Source/WebKit/WPEPlatform/wpe/drm/WPEDRMSeat.cpp

```diff
@@ -182,6 +182,18 @@ void Seat::processEvent(struct libinput_event* event)
     case LIBINPUT_EVENT_KEYBOARD_KEY:
         handleKeyEvent(libinput_event_get_keyboard_event(event));
         break;
+    case LIBINPUT_EVENT_TOUCH_DOWN:
+        handleTouchDownEvent(libinput_event_get_touch_event(event));
+        break;
+    case LIBINPUT_EVENT_TOUCH_UP:
+        handleTouchUpEvent(libinput_event_get_touch_event(event));
+        break;
+    case LIBINPUT_EVENT_TOUCH_MOTION:
+        handleTouchMotionEvent(libinput_event_get_touch_event(event));
+        break;
+    case LIBINPUT_EVENT_TOUCH_CANCEL:
+        handleTouchCancelEvent(libinput_event_get_touch_event(event));
+        break;
```

Comment:
I am surprised that this does not handle `LIBINPUT_EVENT_TOUCH_FRAME`: in order to support multitouch input, multiple events may be received at the same time, and the “frame” marker indicates when all the simultaneous multitouch events have been picked. 

Or are you leaving multitouch support for a follow-up patch?

Also, since libinput 1.19 pinch, swipe, and hold gestures (`LIBINPUT_EVENT_GESTURE_*`) are directly recognized by libinput, we may want to use those e.g. touch-hold to trigger the context menu.